razorpay/client.py

```diff
@@ -102,66 +140,6 @@ def _update_request(self, data, options):
         if 'headers' in options:
             options['headers']['Content-type'] = 'application/json'
         else:
-            options['headers'] = {'Content-type' : 'application/json'}
+            options['headers'] = {'Content-type': 'application/json'}
```

Comment:
then it will fail, as we are sending dictionary object in invoice and without this header our req will fail